packages/shell/esm-app-shell/webpack.config.js

```diff
@@ -157,10 +157,24 @@ module.exports = (env, argv = {}) => {
           type: "asset/source",
         },
         {
-          test: /\.(j|t)sx?$/,
+          test: /\.(js|jsx)$/,
```

Comment:
Personally, I find this easier to read...

```suggestion
          test: /\.jsx?$/,
```